packages/cli/lib/projects.js

```diff
@@ -55,13 +55,16 @@ const SPINNER_STATUS = {
 };
 
 const writeProjectConfig = (configPath, config) => {
+  let success = true;
   try {
     fs.ensureFileSync(configPath);
     fs.writeFileSync(configPath, JSON.stringify(config, null, 2));
     logger.debug(`Wrote project config at ${configPath}`);
   } catch (e) {
-    logger.error(`Could not write project config at ${configPath}`);
+    logger.debug(e);
+    success = false;
```

Comment:
Nit: we could `return false` here and `return  true` on line 67 and eliminate the `success` variable

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1115/commits/1031c18d71632dead15b6082e22b6366be615501